openpype/hosts/nuke/startup/custom_write_node.py

```diff
@@ -53,24 +58,74 @@
 }
 
 
+class WriteNodeKnobSettingPanel(nukescripts.PythonPanel):
+    """ Write Node's Knobs Settings Panel """
+    def __init__(self):
+        nukescripts.PythonPanel.__init__(self, "Set Knobs Value(Write Node)")
+
+        knobs_value, _ = self.get_node_knobs_setting()
+        # create knobs
+
+        self.typeKnob = nuke.Enumeration_Knob(
```

Comment:
Here the name is also not very real. `typeKnob` should in fact be `preset_selector`. Also the consistency of names is not good. Lets keep all variable names in snake_case style. Please use some linter which will catch this in your IDE.